data/brands/shop/supermarket.json

```diff
@@ -3175,7 +3175,6 @@
       "tags": {
         "brand": "IGA",
         "brand:wikidata": "Q5970945",
-        "name": "IGA",
```

Comment:
This will result in new POIs with this brand having no name by default. Is it incorrect or only suboptimal to tag it with a name of “IGA”?

To keep “IGA” as the default name but allow the user to easily modify an individual location’s name, keep the `name` but add a `preserveTags` to the entry instead:

https://github.com/osmlab/name-suggestion-index/blob/8dcc09b2265644e230781b8c60b6a713985f09f9/data/brands/shop/supermarket.json#L850

This will keep the Name field unlocked and suppress the warning that currently occurs when you change the name away from “IGA” to something more specific. This change would be equally applicable to IGA in the United States: https://github.com/osmlab/name-suggestion-index/issues/5500#issuecomment-943095957.

Reply:
That is the intended result.

Reply:
> This change would be equally applicable to IGA in the United States: https://github.com/osmlab/name-suggestion-index/issues/5500#issuecomment-943095957.

dda2181cd3ab593016f43953f0173fb6067c9379 applies the same to the IGA entry for North America.